.babelrc

```diff
@@ -1,7 +1,7 @@
 {
   "presets": [
-    ["@babel/env", { "modules": false }],
-    "@babel/stage-3",
+    ["@babel/env", { "loose": true, "modules": false }],
```

Comment:
Humm, I have tried that on Material-UI side, it was making the browser tests going crazy.

Reply:
I saw mui behaviour for the first time. In other projects it worked pretty well. And here it works good,

Reply:
How can we know without a test suite?

Reply:
Oh, unless it was a phantomJS issue only

Reply:
Ok, I think that we should wait to enable the loose mode on Material-UI first before trying to rely on the loose mode.

Reply:
Probably.

It's safe in a dozens of projects I maintain. Mui problem may be to different stuff.

Reply:
There is also one conceptual issue with the loose mode. The implementation doesn't match the specification. You might very well endup in a case where you rely on some edge case of the transpilation that will no longer work once you remove Babel to use the native feature.

Reply:
By the way, we can remove PhantomJS and replace it with ChromeHeadless, puppeteer ease the installation overhead.

Reply:
That's why you should test the code without transpilation.